src/styles/main.scss

```diff
@@ -1,3 +1,39 @@
-body {
+@import "./blocks/variables";
+@import "./blocks/header";
+@import "./blocks/card";
+@import "./blocks/stars";
+@import "./blocks/logo";
+
+* {
+  box-sizing: border-box;
   margin: 0;
+  padding: 0;
```

Comment:
don't reset margins and paddings for `*` selector, it is ok to use only for box-sizing